### DISCUSSION TITLE
Intellisense for cake does not work

### DISCUSSION THREAD

**Discussion by BrushkouMatvey** [cake-build/cake#4233] [Extension Q&A]
Intellisense for cake does not work for me, and there is no omniShell select project command
<img width="1088" alt="image" src="https://github.com/cake-build/cake/assets/48620777/e9a1ddbe-62aa-4190-a45d-6817655817f7">

**Comment by nils-a** [comment]
Have a look at https://github.com/orgs/cake-build/discussions/4232

**Reply by BrushkouMatvey** [reply]
<img width="322" alt="image" src="https://github.com/cake-build/cake/assets/48620777/c3398f50-9cd1-4360-b689-204b5a2b800c">
<img width="991" alt="image" src="https://github.com/cake-build/cake/assets/48620777/766791d3-717d-40b3-8014-eafb1a48e26b">
